nautobot/core/constants.py

```diff
@@ -1,262 +1,50 @@
-from collections import OrderedDict
```

Comment:
Consider moving `nautobot.utilities.config` to `nautobot.core.utils.config` instead of the top-level `nautobot.core.config` so as to reduce the likelihood of confusing it with `nautobot.core.settings`?

Reply:
I know we went back and forth in the discussion last week but I thought we landed on

`nautobot.core.[module].utils` instead of `nautobot.core.utils.[module]`

Reply:
The thing is a lot of these don't *have* directly corresponding top-level modules... `nautobot.core.config.utils` isn't any less messy than a top-level `nautobot.core.config`, whereas `nautobot.core.utils.config` at least groups a lot of disparate files under the `.utils` namespace.

For cases where there *is* a top-level module already, I've done my best to follow that (`nautobot.core.api.utils`, `nautobot.core.views.utils`). It's a messy problem to be sure. :-)

Reply:
Since relocating is pain, I'd like to follow a convention if possible. Thankfully this ditches the entire utilities app but `core.utils.config` and `core.views.utils` still seems opaque to me.

Reply:
`core.utils` is still a bit of a catch-all, but I think this is a net improvement (new diffs coming soon):

```
> ls -1 nautobot/core/**/utils.py
nautobot/core/api/utils.py
nautobot/core/forms/utils.py
nautobot/core/graphql/utils.py
nautobot/core/models/utils.py
nautobot/core/testing/utils.py
nautobot/core/views/utils.py
```

```
> ls -1 nautobot/core/utils/
color.py
config.py
data.py
deprecation.py
filtering.py
git.py
logging.py
lookup.py
permissions.py
requests.py
```

So far I haven't been able to come up with good "top-level" `nautobot.core` namespaces to put any of the remainder into. (I made a stab at converting `nautobot.core.filters` to a submodule and moving `nautobot.core.utils.filtering` to `nautobot.core.filters.utils` but that introduced some unpleasant circular dependencies between `nautobot.core.models`, `nautobot.core.forms`, and `nautobot.core.filters`...)